Calibration/HcalCalibAlgos/macros/CalibMonitor.C

```diff
@@ -968,7 +985,8 @@ void CalibMonitor::Loop(Long64_t nmax, bool debug) {
       ++kount5[0];
     }
     bool select = ((cDuplicate_ != nullptr) && (cDuplicate_->doCorr(0))) ? (cDuplicate_->isDuplicate(jentry)) : true;
-    if (!select) {
+    bool reject = (!cRunEx_) ? cRunEx_->exclude(t_Run) : false;
```

Comment:
Could you please verify? `cRunEx_` is a pointer, and it is `false` if it does not exist.
Shouldn't it be:
```suggestion
    bool reject = cRunEx_ ? cRunEx_->exclude(t_Run) : false;
```